content/pages/download.md

```diff
@@ -158,6 +158,17 @@ versions:
 
       **Development snapshots are not intended for live use!** Expect crashes and make sure to back up your Mixxx settings and library before upgrading as explained in the Testing wiki page.
     downloads:
+      - slug: ubuntu
+        os: Ubuntu 18.04 "Bionic Beaver" or later
+        text: |
+          We provide a [PPA on Launchpad](https://launchpad.net/~mixxx/+archive/ubuntu/nightlies) to make installing install the latest development snapshot of Mixxx as easy as possible.
+          Open a terminal, and enter:
+
+              $ sudo add-apt-repository ppa:mixxx/nightlies
+              $ sudo apt update
+              $ sudo apt install mixxx
+
+          Using the PPA ensures that new package versions will be installed automatically with `apt`. Otherwise, you can download individual packages and install them manually.
```

Comment:
```suggestion
          Using the PPA ensures that new package versions will be installed automatically with `apt`.
```

You didn't specify any packages to install manually.

Reply:
You can follow the PPA link and download the package manually.

This is the use-case when one wants a nightly build but not take the risk of being updated with more bleeding edge stuff. The Alternative would be to install Mixxx and remove the PPA afterwards. 

Following the updated makes perfectly sense when on the release PPA but this is different for nighties. 

What do you think? 
* keep the sentence 
* Remove it
* Reword it
*

Reply:
In that case, can you make it a link?

```suggestion
          Using the PPA ensures that new package versions will be installed automatically with `apt`. Otherwise, you can [download individual packages](https://launchpad.net/~mixxx/+archive/ubuntu/mixxx/+packages) and install them manually.
```